.circleci/main.yml

```diff
@@ -738,18 +738,12 @@ jobs:
         type: string
     docker:
       - image: cimg/python:<< parameters.python-version >>-node
```

Comment:
Is it possible to change this to a cimg/node so that we know we're getting Python from Bazel?

Eventually you want to target our CI image, which doesn't have anything except Bazel in it.

Reply:
Oh I see. Since we are using the bazel venv, we aren't even testing the multiple python versions that this test is intending. Let me look into this.

Reply:
There is definitely a way to import multiple pythons in MODULE.bazel, but i'm not sure how to use them.  will have to research that.